### PR TITLE
Fix Christmas tree bounding box

### LINKED ISSUE #22138: Christmas tree's hitbox is completely disjoint.
## Description
image

it's been several days now how has this been unreported

### PR DESCRIPTION
**Pull request by enumerate0**
## About the PR

Christmas tree bounding box was waaaay off from where it should've been.

## Why / Balance

Fixes #22138.

## Technical details

Simply updated the shape bounds for the prototype fixture.

## Media

| Before | After |
| --- | --- |
| 287871487-47f1f8bc-d962-4171-9078-84bd9ef77610 | image |

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- fix: Fixed Christmas tree bounding box

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Decoration/flora.yml [hunk 1]
```diff
@@ -414,7 +414,7 @@
       fix1:
         shape:
           !type:PhysShapeAabb
-          bounds: "-0.8,-1.5,0.8,-0.8"
+          bounds: "-0.5,0.75,0.5,0"
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Decoration/flora.yml:417]: Winding order is left-bottom-right-top so this is out of order.

Comment (enumerate0) [reply Resources/Prototypes/Entities/Objects/Decoration/flora.yml]: @metalgearsloth Apologies. Good now?

**Comment by notquitehadouken** [comment]
give everyone cholera update when

File: Resources/Prototypes/Entities/Objects/Decoration/flora.yml (hunk 1 above)
Comment (metalgearsloth) [reply Resources/Prototypes/Entities/Objects/Decoration/flora.yml]: yeah, also please avoid force-pushing and send screenshot of the new bounds.

**Pull request merged**